draft-ietf-quic-transport.md

```diff
@@ -542,68 +545,138 @@ the server supports, as shown below.
 See {{version-negotiation}} for a description of the version negotiation
 process.
 
+
 ## Cleartext Packets {#cleartext-packet}
 
-Cleartext packets are sent during the handshake prior to key negotiation. A
-Client Cleartext packet contains:
+Cleartext packets are sent during the handshake prior to key negotiation.
+
+All cleartext packets contain the current QUIC version in the version field.
 
-* Octet 0: 0x82
-* Octets 1-8: Connection ID (initial)
-* Octets 9-12: Packet number
-* Octets 13-16: Version
-* Octets 17+: Payload
+The payload of cleartext packets also includes an integrity check, which is
+described in {{QUIC-TLS}}.
 
-Non-Final Server Cleartext packets contain:
 
-* Octet 0: 0x83
-* Octets 1-8: Connection ID (echoed)
-* Octets 9-12: Packet Number
-* Octets 13-16: Version
-* Octets 17+: Payload
+### Client Initial Packet {#packet-client-initial}
 
-Final Server Cleartext packets contains:
+The Client Initial packet has a type octet of 0x82.  It carries the first
+cryptographic handshake message sent by the client.
 
-* Octet 0: 0x84
-* Octets 1-8: Connection ID (final)
-* Octets 9-12: Packet Number
-* Octets 13-16: Version
-* Octets 17+: Payload
+The client populates the connection ID field with randomly selected values,
+unless it has received a packet from the server.  If the client has received a
+packet from the server, the connection ID field uses the value provided by the
+server.
 
-The client MUST choose a random 64-bit value and use it as the initial
-Connection ID in all packets until the server replies with the final Connection
-ID. The server echoes the client's Connection ID in Non-Final Server Cleartext
-packets.  The first Final Server Cleartext and all subsequent packets MUST use
-the final Connection ID, as described in {{connection-id}}.
+The packet number used for Client Initial packets is initialized with a random
+value each time the new contents are created for the packet.  Retransmissions of
+the packet contents increment the packet number by one, see
+({{packet-numbers}}).
```

Comment:
Not for connection-level flow control. I really think the spec should say reset transport state and then suggest that it MAY carry over RTT information. Is there anything else that you want carried over?